web/modules/custom/dpl_loans/src/Form/LoanListSettingsForm.php

```diff
@@ -30,7 +58,7 @@ public function getFormId(): string {
    * {@inheritdoc}
    */
   public function buildForm(array $form, FormStateInterface $form_state): array {
-    $config = $this->config('dpl_loan_list.settings');
+    $config = $this->configService->getConfig();
```

Comment:
Please use `loadConfig` as mentioned earlier. And change conf keys correspondingly.